substrate/frame/salary/src/lib.rs

```diff
@@ -382,6 +393,54 @@ pub mod pallet {
 		}
 	}
 
+	#[pallet::tasks_experimental]
+	impl<T: Config<I>, I: 'static> Pallet<T, I> {
+		#[pallet::task_list(vec![].into_iter())]
+		#[pallet::task_condition(|| {
+			let now = frame_system::Pallet::<T>::block_number();
+			let cycle_period = Pallet::<T, I>::cycle_period();
+			let mut status = Status::<T, I>::get().unwrap();
+			status.cycle_start.saturating_accrue(cycle_period);
+			now >= status.cycle_start
+		})]
+		#[pallet::task_weight(T::DbWeight::get().reads(1))]
+		#[pallet::task_index(0)]
+		pub fn bump_offchain() -> DispatchResult {
+			let mut status = Status::<T, I>::get().ok_or(Error::<T, I>::NotStarted)?;
+			status.cycle_index.saturating_inc();
+			status.cycle_start = frame_system::Pallet::<T>::block_number();
+			status.budget = T::Budget::get();
+			status.total_registrations = Zero::zero();
+			status.total_unregistered_paid = Zero::zero();
+			Status::<T, I>::put(&status);
+
+			Pallet::deposit_event(Event::<T, I>::CycleStarted { index: status.cycle_index });
+			Ok(())
+		}
+	}
+
+	#[pallet::hooks]
+	impl<T: Config<I>, I: 'static> Hooks<BlockNumberFor<T>> for Pallet<T, I> {
+		#[cfg(feature = "experimental")]
```

Comment:
I think we can remove experimental now and get this audited. Thoughts @kianenigma @ggwpez ?